docroot/sites/all/modules/dev/ws_rules_features/modules/positive_feedback_rules/positive_feedback_rules.rules_defaults.inc

```diff
@@ -22,16 +22,16 @@ function positive_feedback_rules_default_rules_configuration() {
       ],
       "DO" : [
         { "mail" : {
-            "to" : "[node:field_member_i_trust:mail]",
-            "subject" : "[Warmshowers.org] You have received member feedback",
+            "to" : [ "node:field-member-i-trust:mail" ],
```

Comment:
You're right - good eye, but yes, that's what features spits out! I'll be suspicious also. 

And a weirdness of this one is that changing the rule did not invalidate the feature or anything.